debian/rules

```diff
@@ -24,4 +24,6 @@ override_dh_install:
 	docker save -o debian/tmp/var/lib/delphix-virtualization/docker-python-2-7-18-slim.tar registry.delphix.com/python:2.7.18-slim
 	docker pull registry.delphix.com/python:3.8-slim
 	docker save -o debian/tmp/var/lib/delphix-virtualization/docker-python-3-8-slim.tar registry.delphix.com/python:3.8-slim
```

Comment:
I'm guessing we still need this since we support plugins built w/ python 2.7 and python 3.8? And btw, is the change to `DockerContants.java` and `DockerManager.java` as follow up change?

Reply:
Yes, the change is to just push the tar to Docker Engine. The changes to Dlpx-app-gate and dvp will be next